### PR TITLE
Account admin made by task

### PR DESCRIPTION
**Pull request by samnangcattor**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,8 @@ gem "rails", "4.2.1"
 gem "mysql2"
 gem "devise"
 gem "bootstrap-sass",       "3.2.0.0"
+gem "faker", "1.4.2"
```
Comment (HorNorin) [inline Gemfile]: move to development

File: config/schedule.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+env :PATH, ENV["PATH"]
+
+set :environment, "development"
+set :output, {:error => "log/cron_error_log.log", :standard => "log/cron_log.log"}
```
Comment (HorNorin) [inline config/schedule.rb]: change from `:error =>` to `error:`

File: config/schedule.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+every 2.minutes do
+  rake "events:fetch"
```
Comment (HorNorin) [inline config/schedule.rb]: I think use rake "admin:create" is better

**Comment by HorNorin** [comment]
ok

**Comment by RathanakSreang** [comment]
Ok

**Pull request merged**